SwiftWisdom/Core/Geometry/Geometry.swift

```diff
@@ -86,6 +86,11 @@ public func random(range: Range<Int>) -> Int {
 }
 
 public func avg(numbers: CGFloat...) -> CGFloat {
+    return avg(numbers)
+}
+
+public func avg(numbers: [CGFloat]) -> CGFloat {
```

Comment:
Is there a reason to have it optional? I'd rather see it return 0 personally.  Again, I might be the minority on this one :smile: